packages/blade/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -805,4 +805,81 @@ WithMultipleDropdowns.args = {
   name: 'design-system',
 };
 
+export const DropdownPerformance = (): React.ReactElement => {
+  const fruits = [
+    'Apples',
+    'Apricots',
+    'Avocados',
+    'Bananas',
+    'Boysenberries',
+    'Blueberries',
+    'Bing Cherry',
+    'Cherries',
+    'Cantaloupe',
+    'Crab apples',
+    'Clementine',
+    'Cucumbers',
+    'Damson plum',
+    'Dinosaur Eggs',
+    'Dates',
+    'Dewberries',
+    'Dragon',
+    'Elderberry',
+    'Eggfruit',
+    'Evergreen',
+    'Huckleberry',
+    'Entawak',
+    'Fig',
+    'Farkleberry',
+    'Finger Lime',
+    'Grapefruit',
+    'Grapes',
+    'Gooseberries',
+    'Guava',
+    'Honeydew melon',
+    'Hackberry',
+    'Honeycrisp Apples',
+    'Indian Prune',
+    'Indonesian Lime',
+    'Imbe',
+    'Indian Fig',
+    'Jackfruit',
+    'Java Apple',
+    'Jambolan',
+    'Kiwi',
+    'Kaffir Lime',
+    'Kumquat',
+    'Lime',
+    'Longan',
+    'Lychee',
+    'Loquat',
+    'Mango',
+    'Mandarin',
+    'Orange',
+    'Mulberry',
+  ];
+
+  console.log('total items', fruits.length);
+  const [clickCount, setClickCount] = React.useState(0);
```

Comment:
Do we need this in the story?

Reply:
Let's keep it, I'll need to pull in changes in the other PR. I'll remove it then.